src/views/objectBrowser.ts

```diff
@@ -541,7 +541,7 @@ export function initializeObjectBrowser(context: vscode.ExtensionContext) {
             })
 
             const messages = Tools.parseMessages(copyResult.stderr);
-            if (messages.findId(`CPF2869`)) {
+            if (messages.length && !messages.findId(`CPF2869`)) {
```

Comment:
Following up with the above comment, we can simply access the length `property` from `messages`.
```suggestion
            if (messages.messages.length && !messages.findId(`CPF2869`)) {
```

Reply:
My lack of knowledge... couldn't make `.length` work, because I didn't duplicate `messages.`...

Of course you're right - I'll fix it.